File: Volume 10 (1000 - 1099)/1066/1066.rb
#!/usr/bin/ruby

par, impar, negativo, positivo = 0, 0, 0, 0
for i in 0...5
    n = gets.to_i
    if n % 2 == 0
        par += 1
    else
        impar += 1
    end
    if n > 0
        positivo += 1
    elsif n < 0
        negativo += 1
    end 
end
puts "%d valor(es) par(es)" % [par]
puts "%d valor(es) impar(es)" % [impar]
puts "%d valor(es) positivo(s)" % [positivo]
puts "%d valor(es) negativo(s)" % [negativo]

File: Volume 10 (1000 - 1099)/1066/1066.c
#include <stdio.h>

int main() {
    int n, i, negativo = 0, positivo = 0, impar = 0, par = 0;
    for (i = 0; i < 5; i++) {
        scanf("%d", &n);
        if (n % 2 == 0) par++;
        else impar++;
        if (n > 0) positivo++;
        else if (n < 0) negativo++;
    }
    printf("%d valor(es) par(es)\n", par);
    printf("%d valor(es) impar(es)\n", impar);
    printf("%d valor(es) positivo(s)\n", positivo);
    printf("%d valor(es) negativo(s)\n", negativo);
    return 0;
}